installer/build-mac-dmg.sh

```diff
@@ -88,23 +88,28 @@ while [ "$(( $(date +%s) - 3600 ))" -lt "$START" ]; do
     notarize_info=$(xcrun altool --notarization-info "$REQUEST_UUID" -u "jonathan@openshot.org" -p "@keychain:NOTARIZE_AUTH")
     echo "$notarize_info"
 
-    pat='Status: (.*)'
+    # Match status (stop at newline)
+    pat='Status: ([^'$'\n'']*)'
     [[ "$notarize_info" =~ $pat ]]
     notarize_status="${BASH_REMATCH[1]}"
+    echo "Notarization Status Found: $notarize_status"
 
     if [ "$notarize_status" != "in progress" ] && [ "$notarize_status" != "" ]; then
```

Comment:
so `$notarize_status` was never `== "in progress"` because the string was "in progress `\n` and another line" Right?